VeroliBGV/BGV008.xml

```diff
@@ -0,0 +1,836 @@
+            <title xml:lang="en" corresp="#title1">Psalter</title>
+            <editor key="MV"/>
+            <editor key="SH"/>
+            <editor key="ABr"/>
+            <funder>Akademie der Wissenschaften in Hamburg</funder>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <pubPlace>Hamburg</pubPlace>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed
+            under the Creative Commons Attribution-ShareAlike 4.0. </p>
+               </licence>
+            </availability>
+            <date>2020-11-14T14:52:12.552+02:00</date>
+         </publicationStmt>
+         
+         <sourceDesc>
+            <p>
+               <cit>
+                  <quote xml:lang="it">
+                     Salterio, seguito dai soliti cantici e dal Weddâse Mâryâm.
+                     16 x 11,6, f. 176 di cui 2 in principio e uno bianco alla fine. Senza data.
+                     Rilegatura in legno non ricoperto. Riposto in una custodia.
+                  </quote>
+                  <bibl>
+                     <ptr target="bm:Zanutto1932Bibliografia"/>
+                     <citedRange unit="page">93</citedRange>
+                     <citedRange unit="item">8</citedRange>
+                  </bibl>
+               </cit>
+            </p>
+         </sourceDesc>
+         
+         <sourceDesc>
+            <msDesc xml:id="ms">
+          
+               <msIdentifier>
+                  <repository ref="INS0453Bibl_Giov_Veroli"/>
+                  <collection/>
+                  <idno>BGV-008</idno>
+                  <altIdentifier>
+                     <idno>Zanutto 8</idno>
+                  </altIdentifier>
+               </msIdentifier>
+          
+               <msContents>
+                  <summary/>
+            
+                   <msItem xml:id="ms_i1">
+                      <locus from="3r" to="175vb"/>
+                     <title type="complete" ref="LIT2701Dawit"/>
+                      <msItem xml:id="ms_i1.1">
+                         <locus target="#3r"/>
+                         <title type="incomplete" ref="LIT4009Naahab"/>
+                         <note>See the transcription in <ref target="#LIT4009Naahab"/>.</note>
+                      </msItem>
+                     <msItem xml:id="ms_i1.2">
+                        <locus from="3r" to="136r"/>
+                        <title type="complete" ref="LIT2000Mazmur"/>
+                        <incipit xml:lang="gez">
+                           <locus target="#3ra"/>
+                           <lb/><hi rend="rubric">ፍካሬ፡ ዘጻድቃን፡ ወዘኃጥአን።</hi>
```

Comment:
Anto has raised an interesting point, which I'll try to summarize here.

- As a rule we type the text as it is written in the manuscript, with all its scribal idiosyncrasies;
- We can correct a scribal error in multiple ways. In case a single letter is mistaken we can encode as follows: `<choice><sic>ዋ</sic><corr resp="MV">ወ</corr></choice>`, optionally indicating the cataloguer as the one who is taking the responsibility of the correction (this will not appear in the printed catalogue).
- There are exceptions to this rule, which are the same applied in critical editions: homophonous alternations (e.g. ዐ/ዓ/አ/ኣ, ሠ/ሰ, ጸ/ፀ) are usually left uncorrected.
- Since it might be not intuitive when we introduce a correction and when we leave the text of the manuscript (even though morphologically or etymologically wrong) as it is, I propose to write a line in the introduction in which we explain this, like it is usually done in critical editions.

Reply:
You are right, Sanne. I'm doing the same and I think this is the best think to do, for many reasons. I was a bit confused with all these plurals written with the first order because what I usually find is an eccess of use of the fourth order when I would expect the first order. But if this a widespread feature, it is fine.

Reply:
> Anto has raised an interesting point, which I'll try to summarize here.
> 
> * As a rule we type the text as it is written in the manuscript, with all its scribal idiosyncrasies;
> * We can correct a scribal error in multiple ways. In case a single letter is mistaken we can encode as follows: `<choice><sic>ዋ</sic><corr resp="MV">ወ</corr></choice>`, optionally indicating the cataloguer as the one who is taking the responsibility of the correction (this will not appear in the printed catalogue).
> * There are exceptions to this rule, which are the same applied in critical editions: homophonous alternations (e.g. ዐ/ዓ/አ/ኣ, ሠ/ሰ, ጸ/ፀ) are usually left uncorrected.
> * Since it might be not intuitive when we introduce a correction and when we leave the text of the manuscript (even though morphologically or etymologically wrong) as it is, I propose to write a line in the introduction in which we explain this, like it is usually done in critical editions.

Perfect, Massimo! I agree with your proposal (I'm sorry I added some more comments before seeing that you already replied; I'm still very much chaotic in using this system. I apologise).

Reply:
No worry at all! It happens when you receive lots of emails for any comment while you're replying in your turn.